Python/sysmodule.c

```diff
@@ -1951,6 +1978,9 @@ _PySys_BeginInit(void)
                                PyDict_GetItemString(sysdict, "displayhook"));
     SET_SYS_FROM_STRING_BORROW("__excepthook__",
                                PyDict_GetItemString(sysdict, "excepthook"));
+    SET_SYS_FROM_STRING_BORROW(
```

Comment:
Your indentation here is different from what's already present here.  While I personally prefer how you've done it, local inconsistency is ugly.

Reply:
True, but unfortunately that's the only way to not break 80 columns (or maybe "the least horrible way").

Reply:
Changing the indentation of the other entries should not be discarded as an option if that allows PEP 7 conformance.

Not ideal, perhaps, but allows local consistency.

Reply:
I'm not sure I want to touch lines outside those related to this PEP.

Reply:
In the interest of local consistency, that could be done after PEP code is landed (assuming acceptance, which sounds likely), as a separate commit. Otherwise, some way to conform would be valuable.